docs/workflow.rst

```diff
@@ -0,0 +1,231 @@
+********
+Workflow
+********
+
+This page provides a brief explanation of the workflow to perform APR calculations with `pAPRika`. For more detail users
+are recommended to go through the tutorials, which details further on how to setup and run APR simulations from start to
+finish.
+
+.. figure :: _static/images/flowchart.png
+   :figwidth: 500px
+   :width: 170px
+   :align: center
+
+   Flowchart of the *pAPRika* workflow for a typical APR simulation.
+
+Structure Preparation
+---------------------
+The starting structure for the APR simulation can be configured with *pAPRika*. The ``Align`` module provides functions
+to shift and orient a structure. For example, we can translate a structure to the origin and then orient the system to
+the :math:`z`-axis by running
+
+.. code ::
+
+    from paprika.align import *
+
+    translated_structure = translate_to_origin(structure)
+    aligned_structure = zalign(translated_structure, ":GST@C1", ":GST@C2")
+
+We use dummy atoms to define the a reaction coordinate for to pull the guest molecule. We can add dummy atoms to a
+structure using the ``Dummy Atoms`` module in *pAPRika*.
+
+.. code ::
+
+    from paprika.dummy import add_dummy
+
+    structure = dummy.add_dummy(structure, residue_name="DM1", z=-6.0)
+    structure = dummy.add_dummy(structure, residue_name="DM2", z=-9.0)
+    structure = dummy.add_dummy(structure, residue_name="DM3", z=-11.2, y=2.2)
+    structure.save("aligned_with_dummy.pdb", overwrite=True)
+
+We can use the ``tleap`` wrapper to combine all of these components to generate `AMBER` topology and coordinate files.
+
+.. code ::
+
+    import paprika.tleap as tleap
+
+    system = tleap.System()
+    system.output_prefix = "host-guest-dum"
+    system.pbc_type = None
+    system.neutralize = False
+
+    system.template_lines = [
+        "source leaprc.gaff",
+        "HST = loadmol2 host.mol2",
+        "GST = loadmol2 guest.mol2",
+        "DM1 = loadmol2 dm1.mol2",
+        "DM2 = loadmol2 dm2.mol2",
+        "DM3 = loadmol2 dm3.mol2",
+        "model = loadpdb aligned_with_dummy.pdb",
+    ]
+    system.build()
+
+
+Defining Restraints
+-------------------
+
+.. figure :: _static/images/restraints.png
+   :figwidth: 550px
+   :align: center
+
+In APR calculations we apply restraints on the host (or protein) and the guest molecules. The restraints can be grouped
+into four categories: (1) *static restraints*, (2) *varying restraints*, (3) *wall restraints* and (4) *positional
+restraints*.
+
+**(1) Static Restraints**
+
+Static restraints do not change during the whole APR process and do not affect the free-energy. We apply static
+restraints on the host (or protein) molecule to define a guest molecule path. The static restraints are composed of
```

Comment:
I'm not sure I agree with "to define a guest molecule path." It's more to orient the host/protein degrees of freedom, right?

Reply:
Yes, you are correct.